doc/ref/api/manual/sampledata.md

```diff
@@ -0,0 +1,82 @@
+# hvplot.sampledata
+
+The `hvplot.sampledata` module provides convenient access to sample datasets for learning, testing, and prototyping with hvPlot.
+
+## Usage
+
+```python
+import hvplot.pandas  # noqa
+import hvplot.xarray  # noqa
+
+# Recommended: Use the hvplot.sampledata namespace directly
+penguins = hvplot.sampledata.penguins("pandas")
+air_temp = hvplot.sampledata.air_temperature("xarray")
+landsat = hvplot.sampledata.landsat_rgb("rioxarray")
+us_states = hvplot.sampledata.us_states("geopandas")
+
+# Alternative: Import the module separately
+from hvplot import sampledata
+penguins = sampledata.penguins("pandas")
+```
+
+## Installation
+
+The `hvsampledata` package is included as a dependency when installing hvPlot with example dependencies. If you installed hvPlot with minimal dependencies, you may need to install it separately:
+
+::::{tab-set}
+
+:::{tab-item} pip
+
+```bash
+pip install hvsampledata
+```
+
+:::
+
+:::{tab-item} conda
+
+```bash
+conda install -c conda-forge hvsampledata
+```
+
+:::
+::::
+
+## Available Datasets
+
+The `hvsampledata` package provides the following datasets, all accessible through `hvplot.sampledata.<function_name>()`:
+
+### Tabular Data
+
+- **`penguins(engine)`** - Palmer penguins dataset (344 rows) with species measurements
```

Comment:
Let's not list all the datasets, that's not fun to maintain 🙃

Reply:
OK. Maybe an alternative would be to use `hvsampledata.__all__`, which would be automatically updated depending on the version.